client/app/scripts/superdesk-archive/module.js

```diff
@@ -57,9 +57,15 @@ define([
             this.count = 0;
         };
 
+        this.items = getItems;
```

Comment:
What's with this public alias for the `getItems` function? Or was it perhaps meant to be a function call, e.g. `getItems()`?